notebook/static/notebook/js/actions.js

```diff
@@ -437,7 +437,18 @@ define(function(require){
                 $('.header-bar').toggle();
                 events.trigger('resize-header.Page');
             }
-        }
+        },
+        'escape': {
```

Comment:
I split my vote 0.3 for Brian, 0.7 for Jon. Esc is naturally what people hit a lot of time when they don't know where they are I would expect `Esc*(many times)`  to bring me to one particular state regardless of where I am.